src/lightning/store/README.md

```diff
@@ -0,0 +1,81 @@
+## Getting Started
+
+- Login to lightning.ai (_optional_) \<-- takes less than a minute.  ⏩
+- Store your models on the cloud \<-- simple call: `to_lightning_cloud(...)`. 🗳️
+- Share it with your friends \<-- just share the "username/model_name" (and version if required) format. :handshake:
+- They download using a simple call: `download_from_lightning_cloud("username/model_name", version="your_version")`. :wink:
+- They load your cool model. `load_from_lightning_cloud("username/model_name", version="your_version")`. :tada:
+- Lightning :zap: fast, isn't it?. :heart:
+
+## Usage
+
+**Storing to the cloud**
+
+```python
+from lightning.store import to_lightning_cloud
+from sample.model import LitAutoEncoder, Encoder, Decoder
+
+# Initialize your model here
+autoencoder = LitAutoEncoder(Encoder(), Decoder())
+
+# Pass the model object:
+# No need to pass the username (we'll deduce ourselves), just pass the model name you want as the first argument (with an optional version):
+# format: `model_name:version` (version can either be latest or combination of digits and full-stops: 1.0.0 for example)
+to_lightning_cloud("unique_model_mnist", model=autoencoder, source_code_path="sample")
+
+# version:
+to_lightning_cloud(
+    "unique_model_mnist",
+    version="1.0.0",
+    model=autoencoder,
+    source_code_path="sample/model.py",
+)
+
+# OR: (this will save the file which has the model defined)
+to_lightning_cloud("krshrimali/unique_model_mnist", model=autoencoder)
+```
+
+You can also pass the checkpoint path: `to_lightning_cloud("model_name", version="latest", checkpoint_path=...)`.
+
+**Downloading from the cloud**
+
+```python
+from lightning.store import download_from_lightning_cloud
+
+download_from_lightning_cloud("krshrimali/unique_model_mnist", output_dir="your_output_dir")
+# OR: (default to lightning_model_storage $HOME/.lightning/lightning_model_store/username/<model_name>/version_<version_with_dots_replaced_by_underscores>/ folder)
+download_from_lightning_cloud("krshrimali/unique_model_mnist")
+```
+
+**Loading model**
+
+```python
+from lightning.store import load_from_lightning_cloud
+
+# from <username>.<model_name>.version_<version_with_dots_replaced_by_underscores>.<model_source_file> import LitAutoEncoder, Encoder, Decoder
+model = load_from_lightning_cloud(
```

Comment:
load / download from lightning cloud are quite similar in name. Would maybe rename the second one since it doesn't actually load from the cloud, it loads from disk

Reply:
Any suggestion?